### PR TITLE
Fix merge conflicts + SoundHelper + fix incorrect names / packages

### PR DESCRIPTION
**Pull request by shedaniel**
- `inGameHud` is wrong
- moved `Overlay` to the gui package instead of the render package because it is gui related and extends DrawableHelper
- Mapped `SmokeRenderer`, `SoundHelper`
- Moved `PlayerInventoryScreen` to client package

P.S. using `save as` will remove duplicate mappings introduced with merge conflicts like `Packet` and `AbstractPacket` for `ki`, etc

### CODE REVIEW

File: mappings/net/minecraft/client/Minecraft.mapping [hunk 1]
```diff
@@ -46,5 +48,12 @@ CLASS net/minecraft/client/Minecraft net/minecraft/client/Minecraft
 	METHOD k tick ()V
 	METHOD l hasLevel ()Z
 	METHOD m switchDimension ()V
+	METHOD o getDebugFirstLine ()Ljava/lang/String;
+	METHOD p getDebugSecondLine ()Ljava/lang/String;
+	METHOD q getDebugForthLine ()Ljava/lang/String;
```
Comment (valoeghese) [inline mappings/net/minecraft/client/Minecraft.mapping]: getDebugFourthLine

File: mappings/net/minecraft/client/Minecraft.mapping [hunk 2]
```diff
@@ -1,6 +1,7 @@
 CLASS net/minecraft/client/Minecraft net/minecraft/client/Minecraft
 	FIELD B soundHelper Lyo;
 	FIELD J running Z
+	FIELD K fpsChunkUpdatesText Ljava/lang/String;
```
Comment (Blayyke) [inline mappings/net/minecraft/client/Minecraft.mapping]: yarn calls this `fpsDebugString`, which is a bit shorter.

File: mappings/net/minecraft/client/options/GameOptions.mapping [hunk 3]
```diff
@@ -1,5 +1,8 @@
 CLASS kv net/minecraft/client/options/GameOptions
+	FIELD A thirdPersonPerspective Z
```
Comment (Blayyke) [inline mappings/net/minecraft/client/options/GameOptions.mapping]: Hmm maybe this could just be `thirdPerson`, though this name is also okay.

**Comment by shedaniel** [comment]
Alright fixed.

**Pull request merged**